packages/client/graph-scaffolder/src/core/renderer.ts

```diff
@@ -429,13 +436,15 @@ export abstract class Renderer<V, E> extends EventEmitter {
 			return false;
 		}
 
-		function nodeDragStart(evt: any): void {
+		function nodeDragStart(evt: d3.D3DragEvent<any, any, any>): void {
 			evt.sourceEvent.stopPropagation();
 
 			// @ts-ignore: D3 "this"
 			node = d3.select(this) as D3SelectionINode<V>;
 
-			if (evt.sourceEvent.shiftKey) {
+			renderer.isDragEnabled = d3.select(evt.sourceEvent.target).classed(dragSelector ?? 'no-drag');
```

Comment:
I don't understand why `no-drag` is introduced here - what are you trying to do?

Reply:
so what im trying to do here is use 'no-drag' as a default selector for disabling drag

Reply:
I think if the idea is to opt-in, then there shouldn't be any defaults - you should explicitly acknowledge that you want the feature.